README.md

```diff
@@ -1,31 +1,59 @@
-### FanzPlay
+# FanzPlay
 
 ## Overview
+
 *Changing the way we play*
 
 FanzPlay is a mobile application that allows fans to compete in live trivia matches during their favorite team’s sporting events. Whether you’re watching the game at home or in the stadium, you can compete against opposing teams' fans for the chance to win bragging rights and prizes from sponsors.
 
-Fans can play the game by answering trivia questions, specific to every matchup, on their screens before the time runs out. Answering correctly can help your team beat the opposing team, and the leaderboard displays which team is winning. FanzPlay provides a fun and engaging experience for every type of fan!! 
+Fans can play the game by answering trivia questions, specific to every matchup, on their screens before the time runs out. Answering correctly can help your team beat the opposing team, and the leaderboard displays which team is winning. FanzPlay provides a fun and engaging experience for every type of fan!!
 
-Visit our project website [here!](https://tarheels.live/comp523teami/)
+Visit our project website [here](https://tarheels.live/comp523teami/)!
 
 ## Platform Components
-This project was made using TypeScript as the main programming language, React Native for the frontend technology, Firebase for our backend technology, and Expo for our developement environent. Our final product combines the two apps created by the previous team for fans and admins. 
 
-**To be added: We also need some explanation of the high-level module structure of the code. The information to be conveyed is this
+This project was made using TypeScript as the main programming language, React Native for the frontend technology, Firebase for our backend technology, and Expo for our development environment. Our final product combines the two apps created by the previous team for fans and admins.
+
+You can find information about the structure and design of our app in our [Design Document](https://tarheels.live/comp523teami/architecture-diagram/)!
+
+## Accessing and Running FanzPlay
+
+For detailed information on how to run the app, please refer to both the User Manual and the Adminstrator Manual on our app website in the section of [Documentation Plan](https://tarheels.live/comp523teami/documentation-plan/).
+
+You can also refer to your [Handoff Plan](https://tarheels.live/comp523teami/handoff-plan/) which includes helpful videos on how to access and run the app.
+
+## Implementation details
```

Comment:
This is a bit nitpicky but would we be able to capitalize "details"?

Reply:
Should be fixed now